packages/@aws-cdk/aws-rds/README.md

```diff
@@ -41,22 +41,13 @@ const cluster = new rds.DatabaseCluster(this, 'Database', {
 });
 ```
 
-To use a specific version of the engine
```

Comment:
I don't mind all these changes but from the PR description I can't tell whether they were intentional or whether this was an accidental commit.

Reply:
Fair; should have mentioned this in the PR description. The version-less versions of these APIs -- the ones the docs were showing as the first example -- are `@deprecated`, so I took the opportunity to remove usages of them from the README.